File: hammer-permission/src/main/java/dev/vini2003/hammer/permission/registry/common/HPEvents.java
package dev.vini2003.hammer.permission.registry.common;

import dev.vini2003.hammer.permission.api.common.manager.RoleManager;
import net.fabricmc.fabric.api.networking.v1.ServerPlayConnectionEvents;

public class HPEvents {
	public static void init() {
		ServerPlayConnectionEvents.JOIN.register(new RoleManager.JoinListener());
	}
}

File: hammer-permission/src/main/kotlin/dev/vini2003/hammer/permission/registry/common/HPEvents.kt
package dev.vini2003.hammer.permission.registry.common

import dev.vini2003.hammer.permission.common.manager.RoleManager
import net.fabricmc.fabric.api.networking.v1.ServerPlayConnectionEvents

object HPEvents {
	fun init() {
		ServerPlayConnectionEvents.JOIN.register(RoleManager.ServerPlayConnectionEventsJoinListener)
	}
}